### PR TITLE
Reconciled PR #313

### PR DESCRIPTION
**Pull request by jimklimov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +24,13 @@ the command is targeted at one dataset and impacts it and all its children,
 allowing to get a consistent point-in-time set of snapshots across multiple
 datasets.
 
-Zetup Inztructionz
+Compilation Instructions
```
Comment (oetiker) [inline README.md]: that would be

Complilation Inztructionz

Comment (jimklimov) [reply README.md]: I think that spelling what was the original PR aimed to "fix", but fine with me ;)

Comment (oetiker) [reply README.md]: :-) humorlesbunch

**Comment by oetiker** [comment]
thanks

**Pull request merged**